packages/react-client/src/ReactFlightClient.js

```diff
@@ -163,6 +163,7 @@ type BlockedChunk<T> = {
   status: 'blocked',
   value: null | Array<(T) => mixed>,
   reason: null | Array<(mixed) => mixed>,
+  intermediaryValue?: mixed,
   _response: Response,
   _children: Array<SomeChunk<any>> | ProfilingResult, // Profiling-only
```

Comment:
This is not "Profiling-only" anymore. I'm not sure this is actually reliable enough. It's more of a hack for profiling which is optional if it works or not.

Reply:
I think we can keep `_children` profiling-only. It was not necessary for this fix to remove the two guards around `initializingChunk._children`. I reverted that.